diff --git a/node_modules/react-native-pjsip/ios/RTCPjSip/PjSipModule.m b/node_modules/react-native-pjsip/ios/RTCPjSip/PjSipModule.m
index e4863dd..50b545d 100755
--- a/node_modules/react-native-pjsip/ios/RTCPjSip/PjSipModule.m
+++ b/node_modules/react-native-pjsip/ios/RTCPjSip/PjSipModule.m
@@ -4,6 +4,7 @@
 #import <React/RCTBridge.h>
 #import <React/RCTEventDispatcher.h>
 #import <React/RCTUtils.h>
+#import <VialerPJSIP/pjsua.h>
 
 @interface PjSipModule ()
 
@@ -21,6 +22,7 @@ - (dispatch_queue_t)methodQueue {
 
 - (instancetype)init {
     self = [super init];
+    pj_activesock_enable_iphone_os_bg(PJ_FALSE);
     return self;
 }
 
